### DISCUSSION TITLE
Subtitle Hilang dalam output ggplot

### DISCUSSION THREAD

**Discussion by AGelfP** [teamalgoritma/community#548] [Q&A] [answered]
Jadi saya sedang menyiapkan interactive plot menggunakan ggplot. Sejauh ini kode sudah menunjukkan output yang sesuai dengan gambaran saya, namun ada sebuah kendala: subtitle tidak muncul di output. Terlampir berikut adalah screenshot kode dan hasil output.
![image](https://user-images.githubusercontent.com/122977451/219874014-f78f71fd-0115-4098-a5ce-8e54c11e084f.png)
Mohon bantuannya mengenai bagian yang salah, terima kasih

**Comment by VictorNugraha** [comment, marked as answer]
Dear @AGelfP,

Hal tersebut dikarenakan library plotly belum menyediakan fiture yang dapat mengtransformasi subtitle pada plot statis menjadi plot interactive. 

Berikut kamu sertakan sebuah referensi yang harapannya dapat membantu, dalam membuat substile pada plot interactive.

https://datascott.com/blog/subtitles-with-ggplotly/

Best Regards,

Team Algoritma

**Reply by AGelfP** [reply]
Halo Kak @VictorNugraha, saya udah coba nih kode yang Kakak referensikan, tapi sepertinya masih belum bisa ya. Terlampir berikut screenshot kode saya:
![image](https://user-images.githubusercontent.com/122977451/220094554-a712bab2-cc81-40da-8e4a-62195a407e28.png)

Apakah ada kemungkinan ini masalah di urutan kode?

**Reply by AGelfP** [reply]
maaf Kak, sudah tersolve sekarang, penulisan yang bekerja di saya seperti ini
![image](https://user-images.githubusercontent.com/122977451/220104553-eb2c3b96-0867-403e-bed5-e46ae953fc28.png)
